src/components/feedbackAndNotifs/MatchSnackbar.js

```diff
@@ -1,35 +1,53 @@
-import * as React from 'react';
-import Stack from '@mui/material/Stack';
-import Button from '@mui/material/Button';
-import Snackbar from '@mui/material/Snackbar';
-import MuiAlert from '@mui/material/Alert';
-
+import * as React from "react";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import Snackbar from "@mui/material/Snackbar";
+import MuiAlert from "@mui/material/Alert";
+import { Typography, IconButton, Icon } from "@mui/material";
+import NextPlanIcon from "@mui/icons-material/NextPlan";
+import ChatBubbleOutlineIcon from "@mui/icons-material/ChatBubbleOutline";
+import { Chat } from "@mui/icons-material";
 const Alert = React.forwardRef(function Alert(props, ref) {
   return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
 });
 
-export default function MatchSnackbar({match}) {
-  const [open, setOpen] = React.useState(match);
-
-  React.useEffect(()=> {
-    console.log(open + "is currently open")
-  })
-
-
+export default function MatchSnackbar({ match, setMatch, advanceQ }) {
   const handleClose = (event, reason) => {
```

Comment:
It is okay that user could only advence if they closed the match pop up.... let's talk about it later but looks good!